### DISCUSSION TITLE
[New Feature] Make it easier to trigger an existing/pre-placed unit to do something

### DISCUSSION THREAD

**Discussion by CCTiberius** [Vinifera-Developers/Vinifera#1497] [Feature Requests]
### Description:
While technically possible already, the way this is done is kinda awkward and not always reliable. It would be so much nicer if you could somehow specify a unit or attach the trigger directly to them, so you are guaranteed that this specific unit gets started with the script.

### Possible Implementation:
The most user-friendly would be a trigger that directly starts a script on the unit(s) it gets attached to, instead of having to first recruit it into a team. But even if the team part is necessary, it would be very nice to have a trigger that just recruits whatever unit(s) it gets attached to.

**Comment by Crimsonum** [comment]
The Group number should be sufficient as-is, given that you can have practically infinite number of groups. You can also change the Group number of any units via triggers. Is there a specific circumstance where Group is not viable?
